building/target_platforms/target_platforms_updates.adoc

```diff
@@ -95,26 +98,41 @@ If your requirement is not available in an update site (p2 repository), eg. only
 
 Non-p2 artifacts will be added to https://github.com/jbosstools/jbosstools-locus[JBoss Tools Locus] as p2 artifacts, so they can be included in the target platform.
 
-== Create a build.xml to mirror 3rd-party repository
 
-First, clone the http://github.com/jbosstools/jbosstools-download.jboss.org[jbosstools-download.jboss.org] Git repository. Then browse to the `jbosstools/updates/requirements` folder. That's where you'll create your `build.xml`.
+== Determine if dependency is mirrored
+
+The next step to requesting a new bundle be added to the target platform is to determine whether the bundle is already mirrored. JBoss Tools mirrors our requirements at http://download.jboss.org/jbosstools/updates/requirements/
+
+If your dependency is hosted at eclipse.org, there's a good chance that the bits are already mirrored. You will want to look in http://download.jboss.org/jbosstools/updates/requirements/mars or substituting the name of whichever release is currently being targeted in place of mars (luna, mars, neon, etc). 
+
+Some projects hosted at eclipse.org may not be present in this location for any number of reasons, or their versions may not be the version you require. In that case, your requirement may have its own folder in http://download.jboss.org/jbosstools/updates/requirements. One example is the `Targeted Management` plugins, which have their own folder named `tm` in the requirements folder of download.jboss.org.  You may need to be creative while looking for your dependencies if your requirement is a subproject of some other larger project. Unfortunately, there's no easy way to determine this definitively, and you may need to work with the build team to locate your requirement. 
+
+If your requirement is not already mirrored, you will need to create a build.xml to mirror the 3rd-party repository. If your dependency is already mirrored, you can skip this next section, and proceed directly to adding it to the target platform. 
```

Comment:
no, this is a task for releng.  not something those opening TP requirements need to bother with.

once we have removed the build.xml requirement and can use something much more managable we can look at requiring it from those opening a TP requirement.